taxi/views.py

```diff
@@ -34,6 +35,27 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    template_name = "taxi/manufacturer_form.html"
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
+    template_name = "taxi/manufacturer_form.html"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
+class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Manufacturer
+    fields = "__all__"
+    template_name = "taxi/manufacturer_confirm_delete.html"
```

Comment:
This is redundant

Reply:
review the video: https://www.youtube.com/watch?v=lvdC3vB9ncs&t=300s&ab_channel=Mateacademy
timeline: 5:00